src/network/netdev/wireguard.c

```diff
@@ -494,6 +494,9 @@ static int wireguard_decode_key_and_warn(
                 return 0;
         }
 
+        if (!streq(lvalue, "PublicKey"))
```

Comment:
unrelated to this, but I just noticed this belatedly: I think this function's "ret" parameter could benefit from being changed to `uint8_t ret[static WG_KEY_LEN]`

Reply:
Done.